taf/api/keystore.py

```diff
@@ -66,6 +81,12 @@ def generate_keys(keystore: Optional[str], roles_key_infos: str) -> None:
     Returns:
         None
     """
+    if keystore is None:
```

Comment:
There is a `load_signing_keys` function, where we attempt to load keys from keystore. We should extend that function too and try to find the default keystore location if keystore location is not specified. There is also `setup_roles_keys`. We could attempt to find the default keystore location there as well